understanding/accessible-authentication-minimum.html

```diff
@@ -125,203 +122,122 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
                   </p>
                   
                </section>
                <section id="copy-paste">
                   
-                  <h3>Copy and paste</h3>
+                  <h3>コピー＆ペースト</h3>
                   
-                  <p>Copy and paste can be relied on to avoid transcription. Users can copy their login
-                     credentials from a local source (such as a standalone third-party password manager)
-                     and paste it into the username and password fields on a login form, or into a web-based
-                     command line interfaces asking for a password. Blocking people from pasting into authentication
-                     fields, or using a different format between the copied text and the input field (for
-                     example, "Enter the 3rd, 4th, and 6th character of your password"), would force the
-                     user to transcribe information and therefore fail this criterion, unless another method
-                     is available.
+                  <p>転記を避けるには、コピー＆ペーストを利用できる。利用者は、ローカルソース (例えば、スタンドアロンのサードパーティパスワードマネージャー) からログイン資格情報をコピーし、ログインフォームのユーザ名及びパスワードのフィールドに貼り付ける、又はパスワードを要求するウェブベースのコマンドラインインターフェースに貼り付けることができる。認証フィールドに貼り付けることをブロックする、又はコピーしたテキストと入力フィールドとの間で異なる形式を使用すると (例えば、「パスワードの 3 番目、4 番目、および 6 番目の文字を入力してください」)、利用者は情報を転記する必要が生じるため、別の方法が利用できない限り、この基準に失敗する。
                   </p>
                   
                </section>
                <section id="two-factor-authentication">
                   
-                  <h3>Two-factor authentication systems (verification codes)</h3>
+                  <h3>2 要素認証システム (検証コード)</h3>
                   
-                  <p>Beyond usernames and passwords, some sites may use two-factor authentication, asking
-                     the user to enter a verification code (also called a passcode or one-time password).
-                     A service that requires <em>manual</em> transcription of a verification code is not compliant. As with usernames and passwords,
-                     it must be possible for a user to at least paste the code (such as from a standalone
-                     third-party password manager, text message application, or software-based security
-                     key), or to allow user agents to fill in the fields automatically.
+                  <p>一部のサイトでは、ユーザ名とパスワード以外に 2 要素認証を使用し、利用者に確認コード (パスコード又はワンタイムパスワードとも呼ばれる) の記入を求める場合がある。検証コードを<em>手作業で</em>転記する必要があるサービスは準拠していない。ユーザ名とパスワードと同様に、利用者が少なくとも (スタンドアロンのサードパーティのパスワードマネージャー、テキストメッセージアプリケーション、ソフトウェアベースのセキュリティキーなどから) そのコードを貼り付ける、又はユーザエージェントがフィールドに自動的に記入できるようにする必要がある。
```

Comment:
verification code の訳が「検証コード」と「確認コード」との間で揺れているので、「検証コード」で統一してみました。

```suggestion
                  <p>一部のサイトでは、ユーザ名とパスワード以外に 2 要素認証を使用し、利用者に検証コード (パスコード又はワンタイムパスワードとも呼ばれる) の記入を求める場合がある。検証コードを<em>手作業で</em>転記する必要があるサービスは準拠していない。ユーザ名とパスワードと同様に、利用者が少なくとも (スタンドアロンのサードパーティのパスワードマネージャー、テキストメッセージアプリケーション、ソフトウェアベースのセキュリティキーなどから) そのコードを貼り付ける、又はユーザエージェントがフィールドに自動的に記入できるようにする必要がある。
```